pages/team/index.js

```diff
@@ -0,0 +1,14 @@
+const Page = () => <></>;
+
+Page.getInitialProps = ({ res }) => {
```

Comment:
Would use https://nextjs.org/docs/basic-features/data-fetching#getserversideprops-server-side-rendering instead so that the if statement isn't needed.